### PR TITLE
Governance process been actualized.

### PR DESCRIPTION
**Pull request by b0noI**
Changes:
* form for topics proposals for Core maintainers review been added
* merge_rules.json file specified as spruce of truth for the list of maintainers (since it is the file that actually defines permissions)
* responsibilities of the module maintainers are added (as per the last core maintainers meeting)

### CODE REVIEW

File: docs/source/community/governance.rst [hunk 1]
```diff
@@ -130,14 +137,12 @@ The Principles
...
 
-* We will have a nomination form, where anyone in the community can
-  nominate a person to a Module maintainer position
-* Every 3 months, the core maintainers go through the nominations,
-  do light filtering around spam or desk-rejection, and draw up a
-  list of potential nominees.
-* The core maintainers ask the specific module maintainers for more
-  information on the nominee. The information should include the following
-  items:
+* Each module will have it's own process, please conact module maintainers
```
Comment (svekars) [inline docs/source/community/governance.rst:140]:
```suggestion
* Each module has its own process. Please contact module maintainers for more information.
```

File: docs/source/community/governance.rst [hunk 2]
```diff
@@ -130,14 +137,12 @@ The Principles
...
+  however if there is no one, you can file a request to the core maintainers
```
Comment (svekars) [inline docs/source/community/governance.rst:141]:
```suggestion
  However, if there is no process identified, you can file a request to the core maintainers
```

File: docs/source/community/governance.rst [hunk 3]
```diff
@@ -130,14 +137,12 @@ The Principles
...
+  via [this form](https://forms.gle/xNeu1byGMZVHcA2q7). Core maintainers are
```
Comment (svekars) [inline docs/source/community/governance.rst:142]:
```suggestion
  by submitting [this form](https://forms.gle/xNeu1byGMZVHcA2q7). Core maintainers are
```

File: docs/source/community/governance.rst [hunk 4]
```diff
@@ -130,14 +137,12 @@ The Principles
...
+  meeting each 3 months.
```
Comment (svekars) [inline docs/source/community/governance.rst:143]:
```suggestion
  meeting every three months.
```
Should we say once a quarter?

File: docs/source/community/governance.rst [hunk 5]
```diff
@@ -130,14 +137,12 @@ The Principles
...
+* If you using the core maintainers route the information in your reqest
```
Comment (svekars) [inline docs/source/community/governance.rst:144]:
```suggestion
* If you are submitting a request to the core maintainers, the information in your request 
```

File: docs/source/community/governance.rst [hunk 6]
```diff
@@ -130,14 +137,12 @@ The Principles
...
-  list of potential nominees.
-* The core maintainers ask the specific module maintainers for more
-  information on the nominee. The information should include the following
-  items:
+* Each module will have it's own process, please conact module maintainers
+  however if there is no one, you can file a request to the core maintainers
+  via [this form](https://forms.gle/xNeu1byGMZVHcA2q7). Core maintainers are
+  meeting each 3 months.
+* If you using the core maintainers route the information in your reqest 
+  should include the following items:
```
Comment (svekars) [inline docs/source/community/governance.rst:145]:
```suggestion
  must include the following items:
```

**Review by svekars — COMMENTED** [review]
Thanks, some suggestions.

**Review by svekars — APPROVED** [review]
LGTM, but check the linter.

**Review by soumith — APPROVED** [review]

**Pull request closed without merging**